lib/DDG/Spice/Maps/Stay.pm

```diff
@@ -0,0 +1,61 @@
+package DDG::Spice::Maps::Stay;
+# ABSTRACT: Rental objects on the map by Stay22
+# Docs: https://www.stay22.com/embed
+
+use strict;
+use DDG::Spice;
+
+triggers start => (
+    "rental in", 
+    "rental near", 
+    "hotel in",
+    "hotel near",    
+    "hotels in",
+    "hotels near",  
+    "hostel in",
+    "hostel near",    
+    "hostels in",
+    "hostels near",
+    "apartment in",
+    "apartment near",
+    "apartments in",
+    "apartments near",
+    "accommodation in", 
+    "accommodation near",
+    "accommodations in", 
+    "accommodations near"
+);
+triggers startend => (
+    "rental", 
+    "hotel",
+    "hotels",
+    "hostel",
+    "hostels",
+    "apartment",
+    "apartments",
+    "accommodation",
+    "accommodations"
+);
+
+# Badwords array.
+my @skip_words = ('me', 'my city');
```

Comment:
Hi @gaulrobe 
Just checked `hotels near portland, me` query will correctly trigger IA, because Mapbox API returns correct answer on Geocoding request with `portland, me` and Stay22 also return correct map in this case, [check please](https://www.stay22.com/embed/gm?address=portland,%20me)